frontend/src/components/nav/navbar.js

```diff
@@ -56,6 +57,15 @@ class NavBar extends React.Component {
         }
     }
 
+    debounceSearch(func, delay) {
+        let inDebounce;
```

Comment:
"inDebounce" is not very descriptive and adds confusion. It's probably better to name this variable something like "timerId".